src/main/java/project/dao/jdbc/CarDaoJdbc.java

```diff
@@ -0,0 +1,209 @@
+            deleteStatement.setLong(1, id);
+            return deleteStatement.executeUpdate() > 0;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't delete car with id " + id, e);
+        }
+    }
+
+    private void deleteOldDrivers(Car car, Connection connection) throws SQLException {
+        String deleteQuery = "DELETE FROM cars_drivers WHERE car_id=?";
+        PreparedStatement deleteStatement = connection
+                .prepareStatement(deleteQuery);
+        deleteStatement.setLong(1, car.getId());
+        deleteStatement.executeUpdate();
+        deleteStatement.close();
+    }
+
+    private void putDrivers(Car car, Connection connection) throws SQLException {
+        String queryPut = "INSERT INTO cars_drivers (driver_id, car_id)"
+                + "VALUES(?, ?)";
+        PreparedStatement stInsertDrivers = connection
+                .prepareStatement(queryPut);
+        stInsertDrivers.setLong(2, car.getId());
+        for (project.models.Driver driver : car.getDrivers()) {
+            stInsertDrivers.setLong(1, driver.getId());
+            stInsertDrivers.executeUpdate();
+        }
+        stInsertDrivers.close();
+    }
+
+    private Car getCarFromResultSet(ResultSet resultSet) throws SQLException {
+        Car car = new Car();
+        car.setId(resultSet.getObject("car_id", Long.class));
+        car.setName(resultSet.getString("car_model"));
+        car.setManufacturer(getManufacturerById(resultSet.getObject("manufacturer_id",
+                Long.class)));
+        List<Driver> drivers = new ArrayList<>();
+        while (resultSet.next()) {
+            drivers.add(getDriverById(resultSet.getObject("driver_id", Long.class)));
+        }
+        car.setDrivers(drivers);
+        return car;
+    }
+
+    private Manufacturer getManufacturerById(Long id) {
+        String selectQuery = "SELECT * FROM manufacturers WHERE manufacturer_id=?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement selectStatement = connection.prepareStatement(selectQuery)) {
+            selectStatement.setLong(1, id);
+            ResultSet resultSet = selectStatement.executeQuery();
+            Manufacturer manufacturer = new Manufacturer();
+            if (resultSet.next()) {
+                manufacturer.setId(id);
+                manufacturer.setName(resultSet.getString("manufacturer_name"));
+                manufacturer.setCountry(resultSet.getString("manufacturer_country"));
+            }
+            return manufacturer;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get manufacturer with id " + id, e);
+        }
+    }
+
+    private Driver getDriverById(Long id) {
+        String selectQuery = "SELECT * FROM drivers WHERE driver_id=? AND deleted=FALSE";
+        try (Connection connection = ConnectionUtil.getConnection();
```

Comment:
тогда здесь лучше выбирать из таблицы cars_drivers и INNER JOIN drivers, а в сам метод передавать car_id
```suggestion
        String selectQuery = ""SELECT * FROM cars_drivers cd"
                + " INNER JOIN drivers d ON cd.driver_id = d.id"
                + " WHERE cd.car_id = ? AND d.deleted = false";
        try (Connection connection = ConnectionUtil.getConnection();
```